Rubberduck.Core/AutoComplete/AutoCompleteService.cs

```diff
@@ -158,7 +158,8 @@ private bool DeleteAroundCaret(AutoCompleteEventArgs e, IAutoComplete autoComple
                 {
                     var left = code.Substring(0, e.CurrentSelection.StartColumn - 2);
                     var right = code.Substring(e.CurrentSelection.StartColumn);
-                    using (var pane = e.CodeModule.CodePane)
+                    using (var codeModule = e.CodeModule)
```

Comment:
`e`, which is not a COM wrapper, gets passed in into this method. So I do not think it is safe to dispose the component on it.